staff/bernat-casasus/arrays/scripts/sort.js

```diff
@@ -15,6 +15,28 @@ function sort(array, reverse) {
             }
         }
         return result;
+    }else{
+        if(reverse === false || reverse === undefined){
+
+        for(var i = 0; i < long; i++){
```

Comment:
tabula millor, aixi sabras on començar i on acaba una funció.

Reply:
Yeah I will fix it with Alt+Shift+F shortcut in VS